### PR TITLE
fix: close connection on aborted response

### PR DESCRIPTION
**Pull request by ronag**
Fixes: https://github.com/mcollina/undici/issues/15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,8 +32,13 @@ address. The path is discarded.
...
 - `timeout`, the timeout after which a request will time out, in
-  milliseconds. Default:
-  `30000` milliseconds.
+  milliseconds.
+  Default: `30e3` milliseconds (30s).
+
+- `maxAbortedRead`, Maximum number of bytes read after which an
+  aborted response will close the connection. Closing the connection
+  will error other inflight requests in pipeline.
+  Default: `1e9` bytes (1MiB).
```
Comment (ronag) [inline README.md:41]: I'm not very fond of this name for the option. @szmarczak thoughts?

File: test/client.js [hunk 2]
```diff
@@ -763,3 +763,50 @@ test('pipelined chunked POST ', (t) => {
     })
   })
 })
+
+test('aborted GET ', (t) => {
```
Comment (mcollina) [inline test/client.js:767]:
```suggestion
test('aborted GET', (t) => {
```

File: README.md (hunk 1 above)
Comment (szmarczak) [reply README.md]: It sounds good to me. Alternatively `maxAbortBytes`, `maxAbortPayload`.

**Comment by ronag** [comment]
Changed name to `maxAbortedPayload`

**Pull request merged**